calendarpal/src/main/java/org/dmfs/android/calendarpal/operations/EventRelated.java

```diff
@@ -35,30 +31,12 @@
  *
  * @author Marten Gajda
  */
-public final class EventRelated<T> implements InsertOperation<T>
+public final class EventRelated<T> extends DelegatingOperation<T>
 {
 
-    private final Operation<T> mDelegate;
-
-
     public EventRelated(@NonNull RowSnapshot<CalendarContract.Events> event, @NonNull InsertOperation<T> delegate)
     {
-        mDelegate = new Related<>(event, "event_id"/* all supported tables use this name as the foreign key column name */, delegate);
+        super(new Referring<>(event, "event_id",  /* all supported tables use this name as the foreign key column name */ delegate));
```

Comment:
note the comment belongs to `"event_id"` not `delegate`, that's why the comma was to the right of the comment.

Reply:
oh, I don't know how that comma crawled away..  fixed it